packages/mdc-toolbar/README.md

```diff
@@ -1,8 +1,7 @@
 # MDC Toolbar
 
-MDC Toolbar provides a RTL-aware Material Design toolbar component adhering to the
-[Material Design toolbar spec](https://www.google.com/design/spec/components/toolbars.html)
-Toolbars scroll with content by default, but supports fixed on
+MDC Toolbar provides a multiple row container for application navigation, title
```

Comment:
SG, but one grammar nit:

"MDC Toolbar acts as a container for multiple rows containing items such as..."